lib/module.js

```diff
@@ -41,9 +41,16 @@ module.exports = function (moduleOptions) {
       onlyFiles: true
     })
 
+    const toPascal = str => {
```

Comment:
I would suggest moving this helper out of module function (top or bottom) and use function instead of const/arrow.

Reply:
moved